packages/compass-aggregations/src/components/saved-pipelines/save-pipeline-card/save-pipeline-card.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useCallback, useState } from 'react';
+import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
+import { connect } from 'react-redux';
+import {
+  deletePipeline,
+  openPipelineById
+} from '../../../modules/saved-pipeline';
+import {
+  Body,
+  Button,
+  ConfirmationModal,
+  css,
+  cx,
+  Icon,
+  KeylineCard,
+  spacing
+} from '@mongodb-js/compass-components';
+
+const { track } = createLoggerAndTelemetry('COMPASS-AGGREGATIONS-UI');
+
+type SavePipelineCardProps = {
+  id: string;
+  name: string;
+  onOpenPipelineConfirm: (id: string) => void;
+  onDeletePipeline: (id: string) => void;
+};
+
+const cardOuter = css({
+  margin: spacing[2]
+});
+
+const card = css({
+  display: 'flex',
+  alignItems: 'center',
+  padding: spacing[2],
+  '&:hover .controls': {
+    visibility: 'visible'
+  }
+});
+
+const controls = css({
+  flex: 'none',
+  marginLeft: 'auto',
+  display: 'flex',
+  alignItems: 'center',
+  gap: spacing[2],
+  visibility: 'hidden'
+});
+
+const button = css({
+  flex: 'none'
+});
+
+const SavePipelineCard: React.FunctionComponent<SavePipelineCardProps> = ({
```

Comment:
Since we are doing a lot of changes to this file how do you feel about a rename of the file? I find the name `save-pipeline-card` to imply you're saving a pipeline. How about `saved-pipeline-card` to show that the pipeline is already saved that we're opening? Fine as is too.

Reply:
Good one, will rename!

Reply:
Done in https://github.com/mongodb-js/compass/pull/3559/commits/3feb84e699f3a80d9922f0624e5c0c9a70b21cab